test/ad.jl

```diff
@@ -111,20 +123,25 @@ Vlist = ((ℂ^2, (ℂ^3)', ℂ^3, ℂ^2, (ℂ^2)'),
           ℂ[Z2Irrep](0 => 3, 1 => 2)',
           ℂ[Z2Irrep](0 => 2, 1 => 3),
           ℂ[Z2Irrep](0 => 2, 1 => 2)),
-         (ℂ[U1Irrep](0 => 2, 1 => 2, -1 => 2),
-          ℂ[U1Irrep](0 => 3, 1 => 1, -1 => 1),
-          ℂ[U1Irrep](0 => 2, 1 => 2, -1 => 1)',
-          ℂ[U1Irrep](0 => 1, 1 => 2, -1 => 2),
-          ℂ[U1Irrep](0 => 1, 1 => 3, -1 => 2)'),
-         (ℂ[SU2Irrep](0 => 3, 1 // 2 => 1),
-          ℂ[SU2Irrep](0 => 2, 1 => 1),
+         (ℂ[FermionParity](0 => 1, 1 => 1),
+          ℂ[FermionParity](0 => 1, 1 => 2)',
+          ℂ[FermionParity](0 => 2, 1 => 2)',
+          ℂ[FermionParity](0 => 2, 1 => 3),
+          ℂ[FermionParity](0 => 2, 1 => 2)),
+         (ℂ[U1Irrep](0 => 2, 1 => 1, -1 => 1),
+          ℂ[U1Irrep](0 => 1, 1 => 1, -1 => 1),
+          ℂ[U1Irrep](0 => 2, -1 => 1)',
+          ℂ[U1Irrep](0 => 1, 1 => 2),
+          ℂ[U1Irrep](0 => 1, 1 => 1, -1 => 2)'),
+         (ℂ[SU2Irrep](0 => 2, 1 // 2 => 1),
+          ℂ[SU2Irrep](0 => 1, 1 => 1),
           ℂ[SU2Irrep](1 // 2 => 1, 1 => 1)',
-          ℂ[SU2Irrep](0 => 2, 1 // 2 => 2),
+          ℂ[SU2Irrep](1 // 2 => 2),
           ℂ[SU2Irrep](0 => 1, 1 // 2 => 1, 3 // 2 => 1)'))
 
-@testset "Automatic Differentiation with spacetype $(TensorKit.type_repr(eltype(V)))" verbose = true for V in
-                                                                                                         Vlist
-    @testset "Basic Linear Algebra with scalartype $T" for T in (Float64, ComplexF64)
+@timedtestset "Automatic Differentiation with spacetype $(TensorKit.type_repr(eltype(V)))" verbose = true for V in
```

Comment:
What is currently the minimal Julia version? Since Julia 1.8, there is no more benefit from using `@timedtestset` over Julia's `Tests.@testset`. (In fact, the former is a copy of the latter).

Reply:
Currently, TensorKit still supports 1.6. I have nothing against bumping this, or even getting rid of the `@timedtestset`, I just noticed that all other tests still use `@timedtestset`, so I went for uniformity. Let's keep them all as `@timedtestset` for now, and consider removing that dependency in a separate PR?